Automattic/pocket-casts-ios
# [Give Ratings] Add What's New

| 📘 Part of: #1879
|:---:|

Fixes #1965 

This PR adds the What's new announcement for Give Ratings

https://github.com/user-attachments/assets/56c59072-a0f4-42be-a157-21f7caf149ff

## To test

> [!NOTE]
> Make sure you have the `giveRatings` feature on

- CI must be 🟢 
- Run the app
- A new prompt should appear
- Tapping on Got It or Cancel should dismiss the sheet
- If it doesn't, you can add this line into the AppDelegate to reset the `UserDefaults`: `Settings.lastWhatsNewShown = "7.69"`

## Checklist
